src/RpcClient.ts

```diff
@@ -2,25 +2,49 @@ import {
   BlockhashWithExpiryBlockHeight,
   TransactionSignature,
   Commitment,
-  PublicKey,
   AccountInfo,
   GetLatestBlockhashConfig,
   RpcResponseAndContext,
   SignatureResult,
   Blockhash,
   Connection,
   ParsedAccountData,
+  Keypair,
+  PublicKey,
 } from "@solana/web3.js";
 import { DAS } from "./types/das-types";
+import {
+  MetadataArgs,
+  MintCNFTResponse,
+  MintCNFTResult,
+  RequiredMetadataArgs,
+} from "./types";
+import axios from "axios";
+import {
+  existingCollection,
+  getAssetID,
+  initCollection,
+  initTree,
+  loadWallet,
+  mintCollectionCompressedNft,
+  mintCompressedNFT,
+} from "./utils/compression";
 
 export type SendAndConfirmTransactionResponse = {
   signature: TransactionSignature;
   confirmResponse: RpcResponseAndContext<SignatureResult>;
   blockhash: Blockhash;
   lastValidBlockHeight: number;
 };
-import axios from "axios";
-
+// Type for Mint cNFT request
+export type MintCNFTRequest = {
+  metadataArgs: RequiredMetadataArgs;
+  ownerKeypair: string; // Owner keypair -- This will be used as payer as well.
+  collection?: boolean | false; // True/False on whether to mint with a collection.
+  treeKeypair?: string; // Public key of existing tree keypair.
```

Comment:
shouldn't the keypairs be actual keypairs? this is more flexible for people because you can load keypairs via multiple mechanisms